app/views/govuk_publishing_components/components/docs/inverse_header.yml

```diff
@@ -45,30 +41,6 @@ examples:
           } %>
           <p class="publication-header__last-changed">Published 22 April 2016</p>
           <!-- end of example content -->
-  with_breadcrumbs_and_paragraph:
-    data:
-      padding_top: false
-      block: |
-        <!-- example content -->
-          <%= render "govuk_publishing_components/components/breadcrumbs", {
-            collapse_on_mobile: true,
-            breadcrumbs: [
-              {
-                title: "Section",
-                url: "/section"
-              },
-              {
-                title: "Education of disadvantaged children",
-                url: "/section/sub-section"
-              }
-            ],
-            inverse: true
-          } %>
-          <%= render "govuk_publishing_components/components/lead_paragraph", {
-            text: "Schools and academies, further and higher education, apprenticeships and other skills training, student funding, early years.",
-            inverse: true
-          } %>
-          <!-- end of example content -->
   with_link:
```

Comment:
This is actually an oversight on my part when I wrote the card for this, I think I could have suggested that we rename this link variant to:
> With link and paragraph

since the variant we're removing mentions using a paragraph. This one with the link doesn't mention it, even though it also contains a paragraph so it might be good to document that.

Reply:
Thanks Hanna, good suggestion, I've updated the text for the variation example to be "With paragraph and link"